README.md

```diff
@@ -34,6 +34,37 @@ Run the inventory app with a `cloud` profile:
 java -jar target/inventory-api-0.0.1-SNAPSHOT.jar --spring.profiles.active=cloud
 ```
 
+### Use a PostgreSQL database on Kubernetes
+
+Build the Inventory API and push to DockerHub.
+
+```shell script
+pack build -p . projectriff/inventory-api
```

Comment:
I'm using pack v0.4.1 and it's telling to select a builder before it can do anything.
What version are you running?

Reply:
I have `v0.5.0`, but yeas I have a default builder configured

Reply:
I'm using `cloudfoundry/cnb:cflinuxfs3`